controller/modelmanager.h

```diff
@@ -160,6 +163,9 @@ class ModelManager : public QObject
     static FunctionTableModel &getFunctionTableModel();
     static FunctionTableProxyModel &getFunctionTableProxyModel();
     static ActiveFeatureFunctionsModel &getActiveFeatureFunctionsModel();
+    static FunctionWeightsTableModel &getFunctionWeightTableModel();
+    static FunctionWeightProxyModel &getFunctionWeightProxyModel();
+    static FunctionWeightDelegate &getFunctionWeightDelegate();
```

Comment:
delegates hatten wir glaub ich in den anderen Fällen (z.B. FeatureTableDelegate) in der entsprechenden UI Klasse erzeugt und gesetzt, nicht im ModelManager